lib/user.rb

```diff
@@ -0,0 +1,31 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require 'pry'
+
+class User
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @joke = joke
+    @jokes << @joke
+  end
```

Comment:
This last line is what we wanted. We didn't expect to have `@joke` as an instance variable though, so there's no need to hang onto that. `@jokes << joke` is all you needed here.